pkg/crd/known_types.go

```diff
@@ -77,10 +77,7 @@ var KnownPackages = map[string]PackageOverride{
 
 	"k8s.io/apimachinery/pkg/util/intstr": func(p *Parser, pkg *loader.Package) {
 		p.Schemata[TypeIdent{Name: "IntOrString", Package: pkg}] = apiext.JSONSchemaProps{
-			AnyOf: []apiext.JSONSchemaProps{
-				{Type: "string"},
-				{Type: "integer"},
-			},
+			XIntOrString: true,
```

Comment:
the anyOf could be left here for backwards compatibility (in addition to the extension).

Reply:
Fixed!